gui/system/migrations/0046_legacy_ui.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('system', '0045_fromname'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='Advanced',
+            name='adv_legacy_ui',
+            field=models.BooleanField(
+                default=False,
+                help_text='Enable to use legacy UI.',
```

Comment:
```suggestion
                help_text='Enable using the legacy UI.',
```

Reply:
@Mrt134 do you think this says more like that this field should be enabled using the legacy UI rather then that this field enables legacy UI ?

Reply:
Hm, it could be interpreted that way. I'll take another stab at it.